userspace/libscap/scap_zlib.h

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Does this need to be in libscap/ directly or can this be considered an implementation detail of the savefile engine?